### PR TITLE
removing cu_sid and cu_path from pathologic string 

### PR DESCRIPTION
**Pull request by AlanBCole**
removing $`cu_sid` and `$cu_path` from pathologic string and replacing them with the global variables `$base_url` and `$base_path` which are set in drupal core (bootstrap.inc/drupal_settings_initialize())

### CODE REVIEW

File: modules/features/cu_wysiwyg/cu_wysiwyg.module [hunk 1]
```diff
@@ -22,25 +22,27 @@ function cu_wysiwyg_wysiwyg_editor_settings_alter(&$settings, $context) {
...
+  // Build pathologic settings on one path per line
+  $all_paths = join("\n", $paths);
+  return $all_paths;
+}
+
+/**
+ * Implements Pathologic's hook_pathologic_alter().
+ */
+function hook_pathologic_alter(&$url_params, $parts, $settings) {
+  $aPlaceToStop = 'here';
```
Comment (jwfuller) [inline modules/features/cu_wysiwyg/cu_wysiwyg.module:45]: What is this function doing?

Comment (AlanBCole) [reply modules/features/cu_wysiwyg/cu_wysiwyg.module:45]: @jwfuller Sorry! I meant to take that out. I'll do that.

**Comment by jwfuller** [comment]
@cathysnider Could you please spin up a site on dev, create a bunch of kinds of links in WYSIWYG fields, and launch it to test this?

**Comment by cathysnider** [comment]
https://www-dev.colorado.edu/p1c1cc150264/

**Comment by cathysnider** [comment]
WYSIWIG Links are created with the "p1" path:

https://www-dev.colorado.edu/p1c1cc150264/pages

Screen Shot 2020-06-16 at 11 16 58 AM

**Comment by AlanBCole** [comment]
@cathysnider What happens if you create a link with just "node/5", for example?

**Comment by cathysnider** [comment]
@AlanBCole When I type 'node/5' into the Internal Link field, the system adds the p1 value to the link. 

Screen Shot 2020-06-16 at 3 21 54 PM

Screen Shot 2020-06-16 at 3 22 09 PM

**Comment by AlanBCole** [comment]
@cathysnider This might not be a problem as it looks like the p1 value is in the place that the `$base_path` part of the url is. The code uses the `$base_path` so the so called p1 value would show up and be considered a local path. The weird thing that I can not replicate is that the full internal url is not used in the href attribute for the link. Locally, for me, I always get the full url, something like `http://localhost:1234/cu-express-4.1.4/test-faq-page`.

**Comment by AlanBCole** [comment]
@kevincrafts Do you mean that I should still use the `$cu_sid`? I do not know where else I could get that p1 value, if the site has one.

**Comment by kevincrafts** [comment]
Can we redeploy this? @cathysnider

**Pull request merged**